mix.exs

```diff
@@ -37,8 +38,17 @@ defmodule Erlef.MixProject do
       {:phoenix_html, "~> 2.11"},
       {:phoenix_live_reload, "~> 1.2", only: :dev},
       {:gettext, "~> 0.11"},
-      {:jason, "~> 1.0"},
-      {:plug_cowboy, "~> 2.1"}
+      {:jason, "~> 1.1", override: true},
+      {:plug_cowboy, "~> 2.1"},
+      {:nabo, "~> 1.0.1"},
+      {:earmark, "~> 1.2.4"},
+      {:timex, "~> 3.6"}
+    ]
+  end
+
+  defp aliases do
+    [
+      "phx.server": ["compile --force", "phx.server"]
```

Comment:
Is the `compile --force` needed? 🙂

Reply:
This is currently required. The solution we're using for blogging bakes the blog results from parsing, etc. into a module. We ran into some issues where by changes to mark down files were not reflected in dev mode, at least. 
 No worries though, about to switch to our own minimal solution for blogs which will make use of ETS. 😄